app/src/main/java/org/oppia/android/app/policies/PoliciesFragmentPresenter.kt

```diff
@@ -0,0 +1,66 @@
+package org.oppia.android.app.policies
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.android.R
+import org.oppia.android.app.fragment.FragmentScope
+import org.oppia.android.app.model.PoliciesArguments
+import org.oppia.android.app.model.PolicyPage
+import org.oppia.android.app.translation.AppLanguageResourceHandler
+import org.oppia.android.databinding.PoliciesFragmentBinding
+import org.oppia.android.util.parser.html.HtmlParser
+import javax.inject.Inject
+
+/** The presenter for [PoliciesFragment]. */
+@FragmentScope
+class PoliciesFragmentPresenter @Inject constructor(
+  private val htmlParserFactory: HtmlParser.Factory,
+  private val resourceHandler: AppLanguageResourceHandler
+) {
+  private lateinit var binding: PoliciesFragmentBinding
+
+  /** Handles onCreate() method of the [PoliciesFragment]. */
+  fun handleCreateView(
+    inflater: LayoutInflater,
+    container: ViewGroup?,
+    policiesArguments: PoliciesArguments
+  ): View {
+    binding = PoliciesFragmentBinding.inflate(
+      inflater,
+      container,
+      /* attachToRoot= */ false
+    )
+    setUpContentForTextViews(policiesArguments.policyPage)
+
+    return binding.root
+  }
+
+  private fun setUpContentForTextViews(policyPage: PolicyPage) {
+    var privacyPolicyDescription = ""
+    var privacyPolicyWebLink = ""
+    when (policyPage) {
+      PolicyPage.PRIVACY_POLICY -> {
+        privacyPolicyDescription =
+          resourceHandler.getStringInLocale(R.string.privacy_policy_content)
+        privacyPolicyWebLink = resourceHandler.getStringInLocale(R.string.privacy_policy_web_link)
+      }
+      PolicyPage.TERMS_OF_SERVICE -> {
+        privacyPolicyDescription =
+          resourceHandler.getStringInLocale(R.string.terms_of_service_content)
+        privacyPolicyWebLink = resourceHandler.getStringInLocale(R.string.terms_of_service_web_link)
+      }
+      else -> PolicyPage.POLICY_PAGE_UNSPECIFIED
```

Comment:
This makes less sense as an exhaustive when now that we have multiple values to define (I think before we could actually simplify it to be an assignment ``when``, but that's a bit more hacky with two values). Suggest just removing this, instead, since it's not actually doing anything.

Reply:
If I remove it, then its giving suggestion to add else branch. I replaced it with if instead.